src/mlir/ast-utils.h

```diff
@@ -99,6 +102,12 @@ namespace mlir::verona
       InitExpr = peg::str2tag("initexpr"), // = 3935567717
       Member = peg::str2tag("member"), // = 75189616
       Lookup = peg::str2tag("lookup"), // = 4099072514
+      InterpString = peg::str2tag("interp_string"), // TODO
```

Comment:
Please don't add anything to the MLIR side in this PR.

Reply:
See my top-level comment. I am using these in `conversion.cc`. I'm happy to not do that anymore, and use something else.